### PR TITLE
feat file-input: add accept attribute to limit accepted file types

### PR DESCRIPTION
**Pull request by PlayerCode**
Upgrade to lx-file-input to specify accepted file types.

### CODE REVIEW

**Comment by clementprevot** [comment]
@PlayerCode Thank you for your contribution

Just a few things:
- Your commit message doesn't fit the [contributing guidelines](https://github.com/lumapps/lumX/blob/master/CONTRIBUTING.md#-git-commit-guidelines): you should give the scope of your change and the summary (first line) shouldn't be more than 100 characters long. It should be something like this "feat file-input: add accept parameter to limited accepted files"
- Could you please add the new parameter to the documentation of the file-input component in "/modules/file-input/demo/file-input.html" and also add an use case with the new parameter (create a file in "/modules/file-input/demo/includes" and include it in "/modules/file-input/demo/file-input.html")

Except for these points, this is a great addition and it works perfectly.
I will merge this PR as soon as you added the documentation.

Thanks again!

File: modules/file-input/views/file-input.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="input-file">
     <span class="input-file__label">{{ lxFileInput.label }}</span>
     <span class="input-file__filename">{{ lxFileInput.fileName }}</span>
-    <input type="file" class="input-file__input">
+    <input type="file" class="input-file__input" accept="{{lxFileInput.accept}}">
```
Comment (clementprevot) [inline modules/file-input/views/file-input.html]: Could you add some spaces after and before the curly brackets

**Comment by PlayerCode** [comment]
@clementprevot I think it's OK now.

**Comment by clementprevot** [comment]
@PlayerCode One last thing, could you remove the "dist/*" files from your PR to avoid conflicts.
They will be regenerated during the release so you don't need to push them

**Comment by PlayerCode** [comment]
@clementprevot Sorry I thought it needed to be included. It's gone now.

**Comment by clementprevot** [comment]
@PlayerCode Hum sorry, I should have been unclear, I meant to not commit those file, but not really delete them from the repository.
Here, we still have conflicts due to the deletion.
You can leave them, but simply don't commit them in you PR (I have to agree that the process is not very clear with this `dist` folder, we will try to make it easier)

**Comment by PlayerCode** [comment]
@clementprevot, I'm not sure how to do this, I found this command on stackoverflow: `git rm -r --cached dist/` is that what you want?

**Comment by clementprevot** [comment]
You could simply do an interactive rebase, delete your last commit where you removed the files and edit the first commit where you added it

**Comment by clementprevot** [comment]
@PlayerCode I made the cleanup for you and created a new branch/PR.
See #537.

Merging it right away.

Thank you for your contribution :+1:

**Pull request closed without merging**